### ISSUE TITLE
How to add Organization Workflows?

### ISSUE THREAD

**Issue by bazzadp** [peter-evans/create-pull-request#243]
### Subject of the issue

I have a Organization Workflows

### Steps to reproduce

As seen in this workflow: https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/.github/workflows/generate_chapters.yml

I'm using this:
        labels: generate chapters
        project: Web Almanac
        project-column: TODO
The label seems to work, but the project is not applied, and I have to manually apply it in [this Pull Request](https://github.com/HTTPArchive/almanac.httparchive.org/pull/795) which was generated from [this job](https://github.com/HTTPArchive/almanac.httparchive.org/runs/645321256?check_suite_focus=true). Weirdly it does not complain the project can't be found.

I tried replacing the project as org/project and also reseting the column name to None and still it doesn't work.

Any ideas?

(love the Action btw - thanks for creating!)

**Comment by peter-evans** [comment]
Hi @bazzadp 

I think the problem is that your project is actually called `2020`, not `Web Almanac`.

![image](https://user-images.githubusercontent.com/18365890/81134762-b7cdc580-8f90-11ea-9e90-d8c79f928113.png)

Try this:
```diff
    - name: Create Pull Request
      uses: peter-evans/create-pull-request@v2
      with:
        title: Update datestamps and generate chapters
        branch-suffix: timestamp
        commit-message: Generate Chapters
        body: |
            Updated datestamps and regenerated the chapters
            - Please merge today or we'll get new datestamps tomorrow and you'll have to merge again!
            - Auto-generated by [create-pull-request][1] GitHub Action
            [1]: https://github.com/peter-evans/create-pull-request
        labels: generate chapters
-       project: Web Almanac
+       project: 2020
        project-column: TODO
```

**Comment by bazzadp** [comment]
We actually have two projects. A repo-level one:
![image](https://user-images.githubusercontent.com/10931297/81145531-585ebe00-8f6e-11ea-9882-282f597224b7.png)

And an organisation level one:
![image](https://user-images.githubusercontent.com/10931297/81145566-6f9dab80-8f6e-11ea-802a-c8cbecccabd6.png)

Can switch to repo-level projects if this is not possible, but thought I'd ask.

**Comment by peter-evans** [comment]
Oh, I see. It's [this project](https://github.com/orgs/HTTPArchive/projects) at the organisation level. I didn't even notice you could make projects there. I'll need to investigate a bit to see if it's possible.

**Comment by bazzadp** [comment]
Let me know if there's anything I can do to help you (e.g. call APIs you don't have permission for).

But if it's a big deal then don't worry - will just move to a repo level project if needed. Was more raising in case there was an obvious way of doing this that I was missing.

**Comment by peter-evans** [comment]
It's something I've not considered before. It looks possible if I modify the action to support organisation level projects. However, it would most likely mean that you couldn't use the default `GITHUB_TOKEN`. That is not scoped for creating cards at the organisation level. You would need to use a [PAT](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) with `repo` **and** `admin:org` scopes. Would you be happy to do that?

![image](https://user-images.githubusercontent.com/18365890/81149320-f4adb280-8fb8-11ea-8b18-c866346fb298.png)

**Comment by bazzadp** [comment]
To be honest, there's no real reason to use an Organization project so would probably just move to repo project to avoid having to set up a PAT. So wouldn't bother making this change just for me.

**Comment by peter-evans** [comment]
Got it. Thank you for raising the issue. I'll wait for further interest in this functionality before looking at adding it.
Closing this issue for now.

**Comment by bazzadp** [comment]
Ok. Thanks for looking so quickly - and for the brilliant GitHub Action!

**Comment by walexi** [comment]
> Got it. Thank you for raising the issue. I'll wait for further interest in this functionality before looking at adding it.
> Closing this issue for now.

Hi, I need this functionality in my current project - being able to modify org-wide boards.

**Comment by peter-evans** [comment]
I've decided to deprecate `project` and `project-column` inputs in favour of a second step using [create-or-update-project-card](https://github.com/peter-evans/create-or-update-project-card) action. See [create a project card](https://github.com/peter-evans/create-pull-request#create-a-project-card) for usage with create-pull-request action. `project` and `project-column` inputs will be removed in the next major version release.

The main reason for shifting card creation to a separate action is because I realised that the implementation in this action has some weaknesses. Additionally, separating it makes it easier to add further functionality.

@walexi I will try to add that functionality to the new [create-or-update-project-card](https://github.com/peter-evans/create-or-update-project-card) action.
